engine/src/main/java/org/hibernate/validator/internal/cfg/context/TypeConstraintMappingContextImpl.java

```diff
@@ -117,6 +118,7 @@
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
 	public PropertyConstraintMappingContext property(String property, ElementType elementType) {
```

Comment:
I'm surprised by this one? Maybe we should mark this method as deprecated instead?